vcd_cli/user.py

```diff
@@ -0,0 +1,140 @@
+from __future__ import division
+
+import click
+from pyvcloud.vcd.org import Org
+
+from vcd_cli.utils import stderr, restore_session, stdout
+from vcd_cli.vcd import vcd
+
+
+@vcd.group(short_help='work with users in current organization')
+@click.pass_context
+def user(ctx):
+    """Work with users in current organization.
+
+\b
+    Examples
+        vcd user create 'user_name' 'password' 'role_name'
+           create user in the current organization with user_name password and role_name .
+    """
+    if ctx.invoked_subcommand not in [None, 'item']:
+        try:
+            restore_session(ctx)
+        except Exception as e:
+            stderr(e, ctx)
+
+
+@user.command(short_help='create user in current organization')
+@click.pass_context
+@click.argument('user_name',
+                metavar='<user_name>',
+                required=True)
+@click.argument('password',
+                metavar='<password>',
+                required=True)
+@click.argument('role_name',
+                metavar='<role-name>',
+                required=True)
+@click.option('-e',
+              '--email',
+              required=False,
+              default='',
+              metavar='[email]',
+              help='User\'s email address.')
+@click.option('-f',
+              '--full-name',
+              required=False,
+              default='',
+              metavar='[full_name]',
+              help='Full name of the user.')
+@click.option('-D',
+              '--description',
+              required=False,
+              default='',
+              metavar='[description]',
+              help='description.')
+@click.option('-t',
+              '--telephone',
+              required=False,
+              default='',
+              metavar='[telephone]',
+              help='User\'s telephone number.')
+@click.option('-i',
+              '--im',
+              required=False,
+              default='',
+              metavar='[im]',
+              help='User\'s im address.')
+@click.option('-E',
+              '--enabled',
+              is_flag=True,
+              required=False,
+              default=False,
```

Comment:
should we enable the user by default?

Reply:
In create user api, the default value is false. So I thought it is better to have commands option consistent with the API.

Reply:
ok